docs/user/installation_and_configuration.rst

```diff
@@ -301,6 +294,105 @@ In this case you may download SLF4J from http://www.slf4j.org/download.html. Ext
 ``slf4j-log4j12-1.7.7.jar`` and place it in the ``lib/common`` directory of the binary distribution.
 If this conflicts with another SLF4J implementation, you may need to remove it from the ``lib/common`` directory.
 
+Kafka Tools
+^^^^^^^^^^^
+
+.. note::
+
+    These instructions assume the use of Kafka 08 but the instructions are identical from Kafka 9 and 10.
```

Comment:
Technically it's 0.8.x, 0.9.x, and 0.10.x. We use "08", "09", and "10" as shorthand in our package/module names.